src/runtime/vdom/h.ts

```diff
@@ -109,7 +109,7 @@ Empty objects can also be the cause, look for JSX comments that became objects.`
  * @param text possible text content for the node
  * @returns a newly-minted virtual DOM node
  */
-export const newVNode = (tag: string, text: string) => {
+export const newVNode = (tag: string | null, text: string | null): d.VNode => {
```

Comment:
Yeah, that makes sense. I made the change basically because we are explicitly passing `null` to both of these parameters in a few places, e.g. here:

https://github.com/ionic-team/stencil/blob/9ace5cd87cd1ed1380087fedd8d7c53e043afc12/src/runtime/vdom/h.ts#L90-L90

and here:

https://github.com/ionic-team/stencil/blob/9ace5cd87cd1ed1380087fedd8d7c53e043afc12/src/runtime/vdom/h.ts#L44-L44

So I figured the typing for the parameters should be inline with that! I'm going to adjust the types for `VNode` and `ChildNode` to match